packages/react-composites/tests/README.md

```diff
@@ -171,3 +176,7 @@ To debug a particular test,
 When working on a specific hermetic test be sure to use `node scripts/runBrowserTests.mjs -l`. This should be used because all of the live tests will be run after even though you are using `test.only(...)`. This happens because the wrapper script calls playwright multiple times depending on which flags are given.
 
 As well you will need to also use the flags `... -c call` and `... -c chat` to run just the composite tests that you are writing tests for.
+
+## Troubleshooting
+
+1. "ERROR: ENOENT: no such file or directory: node_modules/ts-node/dist/bin.js'" or "react-use-draggable-scroll.js doesn't exist" - you need run `rush update -p` under `/packages/react-composites`
```

Comment:
It may be a good idea to separate the issue descriptions and steps to fix it. 
iOS format example:
https://github.com/Azure/communication-ui-library-ios/blob/1e715877c56d0edb81b1d1637ffac3d068737472/AzureCommunicationUI/sdk/AzureCommunicationUICalling/manual-installation.md#how-to-fix-the-issues

Reply:
Do you mean run `rushx update -p` for running in specific package folder? I think `rush update -p` can be run anywhere in the repo directory

Reply:
tbh I don't remember, but you probably are right. I don't need to specify a folder. removing it now.